syft/frameworks/torch/differential_privacy/pate.py

```diff
@@ -266,3 +278,245 @@ def perform_analysis(teacher_preds, indices, noise_eps, delta=1e-5, moments=8, b
     # print("Data independent bound = " + str(min(data_ind_eps_list)) + ".")
 
     return min(eps_list_nm), min(data_ind_eps_list)
+
+
+def tensors_to_literals(tensor_list):
+
+    """Converts list of torch tensors to list of integers/floats. Fix for not having the functionality which converts list of tensors to tensors
```

Comment:
Change: *Converts list of torch tensors to list of integers/floats. Fix for not having the functionality which converts list of tensors to tensors*

to: *Converts a list of torch tensors to list of integers/floats.
Fix for not having the functionality which converts a list of tensors to tensors.*